ProfilerGUI/Source/Configurator/BoolToCollapsingConverter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Data;
+
+namespace ProfilerGUI.Source.Configurator
+{
+    /// <summary>
+    /// Converts a boolean to either Visibility.Visible or Visibility.Collapsed
+    /// </summary>
+    [ValueConversion(typeof(bool), typeof(Visibility))]
+    public class BoolToCollapsingConverter : IValueConverter
```

Comment:
Use .net internal class BooleanToVisibilityConverter

Reply:
as far as I understood, that converts to invisible, not to collapsed. Can you show me how to use the builtin one to get the collapsed state?

Reply:
ok looks like I misunderstood ;-) removed this